README.md

```diff
@@ -17,78 +15,92 @@
 
 The default [JavaScript tracking snippet](https://developers.google.com/analytics/devguides/collection/analyticsjs/) for Google Analytics runs when a web page is first loaded and sends a pageview hit to Google Analytics. If you want to know about more than just pageviews (e.g. events, social interactions), you have to write code to capture that information yourself.
 
-Since most website owners care about most of the same types of user interactions, web developers end up writing the same code over and over again for every new site they build.
+Since most website owners care about a lot of the same types of user interactions, web developers end up writing the same code over and over again for every new site they build.
 
 Autotrack was created to solve this problem. It provides default tracking for the interactions most people care about, and it provides several convenience features (e.g. declarative event tracking) to make it easier than ever to understand how people are using your site.
 
-The `autotrack.js` library is small (3K gzipped), and includes the following plugins. By default all plugins are bundled together, but they can be included and configured separately as well:
+## Plugins
+
+The `autotrack.js` library is small (6K gzipped), and includes the following plugins. By default all plugins are bundled together, but they can be included and configured separately as well. This table includes a brief description of each plugin; you can click on the plugin name to see the full documentation and usage instructions:
 
 <table>
   <tr>
     <th align="left">Plugin</th>
     <th align="left">Description</th>
   </tr>
   <tr>
-    <td><a href="#eventtracker"><code>eventTracker</code></a></td>
-    <td>Declarative event tracking</td>
+    <td><a href="/docs/plugins/clean-url-tracker.md"><code>cleanUrlTracker</code></a></td>
+    <td>Prevents mutliple different URL paths that point to the same page from appearing in your Google Analytics reports.</td>
```

Comment:
better description? makes it sound like you're removing data from reports when you're actually just making them consistent.